Source/WebCore/platform/graphics/cocoa/MediaPlayerPrivateWebM.mm

```diff
@@ -1353,6 +1371,10 @@ static bool isCopyDisplayedPixelBufferAvailable()
     
     m_decompressionSession->invalidate();
     m_decompressionSession = nullptr;
+    if (m_hasAvailableVideoFrameSemaphore) {
+        m_hasAvailableVideoFrameSemaphore->signal();
```

Comment:
Not sure if I should keep this `signal()` call considering this is being called on the main thread (which would be blocked regardless)

Reply:
I don't think it is possible, but why don't you  `ASSERT(! m_hasAvailableVideoFrameSemaphore)`